python_lib/qss_compiler/examples/TestOps.py

```diff
@@ -0,0 +1,144 @@
+# (C) Copyright IBM 2024.
+#
+# This code is part of Qiskit.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Example python script for generating mlir for a variety of Pulse Ops
+
+from qss_compiler.mlir.ir import InsertionPoint, Location, Module, Context
+from qss_compiler.mlir.ir import IntegerType, F64Type, ComplexType
+from qss_compiler.mlir.ir import DenseElementsAttr
+
+from qss_compiler.mlir.dialects import arith, complex, func
+from qss_compiler.mlir.dialects import pulse, quir
+
+import numpy as np
+
+with Context(), Location.unknown():
+    # register dialects
+    pulse.pulse.register_dialect()
+    quir.quir.register_dialect()
+
+    # declare types
+    i1 = IntegerType.get_signless(1)
+    i32 = IntegerType.get_signless(32)
+    f64 = F64Type.get()
+
+    c64 = ComplexType.get(f64)
+
+    p = pulse.PortType.get()
+    wf = pulse.WaveformType.get()
+    mf = pulse.MixedFrameType.get()
+    f = pulse.FrameType.get()
+
+    # create module
+    module = Module.create()
+    with InsertionPoint(module.body):
+        # create main func op
+        function = func.FuncOp("main", ([], [i1, i1]))
+        function.add_entry_block()
+
+        # create test sequence op
+        seq = pulse.SequenceOp("test_pulse_ops", [p, p, f, mf, c64], [i1, i1])
+        seq.add_entry_block()
+    # define sequence
```

Comment:
```suggestion

    # define sequence
```

Reply:
Updated in https://github.com/openqasm/qe-compiler/pull/276/commits/9393557041fc7ef5c563c97aaaab4f734a4c9b55